Remove `innerText` test directory since duplicate of `html/dom/elements/the-innertext-and-outertext-properties`

Ahmad-S792 opened the pull request:
#### 3ef333bb03db41210430e7d627aca83fd71db1a3
<pre>
Remove `innerText` test directory since duplicate of `html/dom/elements/the-innertext-and-outertext-properties`

<a href="https://bugs.webkit.org/show_bug.cgi?id=273504">https://bugs.webkit.org/show_bug.cgi?id=273504</a>

Reviewed by Tim Nguyen.

These tests were moved to following directory on WPT:

html/dom/elements/the-innertext-and-outertext-properties

So this patch is to remove `innerText` separate directory.

* LayoutTests/imported/w3c/web-platform-tests/innerText/getter-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/getter-tests.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/getter.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/setter-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/setter-tests.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/setter.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/w3c-import.log: Removed.
* LayoutTests/imported/w3c/resources/import-expectations.json:

Canonical link: <a href="https://commits.webkit.org/278280@main">https://commits.webkit.org/278280@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8a41978a2d398cc28e27acb928e3d034c6befc48

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [⏳ 🧪 style ](https://ews-build.webkit.org/#/builders/Style-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 ios ](https://ews-build.webkit.org/#/builders/iOS-17-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac ](https://ews-build.webkit.org/#/builders/macOS-Ventura-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wincairo ](https://ews-build.webkit.org/#/builders/Win-Build-EWS "Waiting in queue, processing has not started yet") 
| [⏳ 🧪 bindings ](https://ews-build.webkit.org/#/builders/Bindings-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 ios-sim ](https://ews-build.webkit.org/#/builders/iOS-17-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 mac-AS-debug ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 wpe-wk2 ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | 
| [⏳ 🧪 webkitperl ](https://ews-build.webkit.org/#/builders/WebKitPerl-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 ios-wk2 ](https://ews-build.webkit.org/#/builders/iOS-17-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-mac ](https://ews-build.webkit.org/#/builders/macOS-Ventura-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-wpe ](https://ews-build.webkit.org/#/builders/WPE-Build-EWS "Waiting in queue, processing has not started yet") | 
| [⏳ 🧪 webkitpy ](https://ews-build.webkit.org/#/builders/WebKitPy-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 ios-wk2-wpt ](https://ews-build.webkit.org/#/builders/iOS-17-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk1 ](https://ews-build.webkit.org/#/builders/macOS-Ventura-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 wpe-skia ](https://ews-build.webkit.org/#/builders/WPE-Skia-Build-EWS "Waiting in queue, processing has not started yet") | 
| [⏳ 🛠 🧪 jsc ](https://ews-build.webkit.org/#/builders/JSC-Tests-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-ios ](https://ews-build.webkit.org/#/builders/iOS-17-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Ventura-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 gtk ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | 
| [⏳ 🛠 🧪 jsc-arm64 ](https://ews-build.webkit.org/#/builders/JSC-Tests-arm64-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 tv ](https://ews-build.webkit.org/#/builders/tvOS-17-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-AS-debug-wk2 ](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Debug-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 gtk-wk2 ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | 
| [⏳ 🧪 services ](https://ews-build.webkit.org/#/builders/Services-EWS "Waiting in queue, processing has not started yet") | [⏳ 🛠 tv-sim ](https://ews-build.webkit.org/#/builders/tvOS-17-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 mac-wk2-stress ](https://ews-build.webkit.org/#/builders/macOS-Ventura-Release-Build-EWS "Waiting in queue, processing has not started yet") | [⏳ 🧪 api-gtk ](https://ews-build.webkit.org/#/builders/GTK-Build-EWS "Waiting in queue, processing has not started yet") | 
| | [⏳ 🛠 watch ](https://ews-build.webkit.org/#/builders/watchOS-10-Build-EWS "Waiting in queue, processing has not started yet") | | [⏳ 🛠 jsc-armv7 ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/7228 "Built successfully and passed tests") | [⏳ 🛠 watch-sim ](https://ews-build.webkit.org/#/builders/watchOS-10-Simulator-Build-EWS "Waiting in queue, processing has not started yet") | | [⏳ 🧪 jsc-armv7-tests ](https://ews-build.webkit.org/#/builders/JSC-ARMv7-32bits-Build-EWS "Waiting in queue, processing has not started yet") | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Can you also remove this line: https://searchfox.org/wubkat/rev/088a3e573b59664dc988291e52900bba6b106b2c/LayoutTests/imported/w3c/resources/import-expectations.json#205 ?

Commit:
Remove `innerText` test directory since duplicate of `html/dom/elements/the-innertext-and-outertext-properties`

https://bugs.webkit.org/show_bug.cgi?id=273504

Reviewed by Tim Nguyen.

These tests were moved to following directory on WPT:

html/dom/elements/the-innertext-and-outertext-properties

So this patch is to remove `innerText` separate directory.

* LayoutTests/imported/w3c/web-platform-tests/innerText/getter-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/getter-tests.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/getter.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/setter-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/setter-tests.js: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/setter.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/innerText/w3c-import.log: Removed.
* LayoutTests/imported/w3c/resources/import-expectations.json:

Canonical link: https://commits.webkit.org/278280@main
